rootfs/api/tests/test_pods.py

```diff
@@ -415,6 +415,8 @@ def test_scale_errors(self, mock_requests):
         body = {'web': -1}
         response = self.client.post(url, body)
         self.assertEqual(response.status_code, 400, response.data)
+        self.assertEqual(response.data, {'detail': "Invalid scaling format: "
+                                         "['Must be greater than or equal to zero']"})
```

Comment:
I agree, but that's out of the scope of this issue. Mind filing an issue for it?